bin/postman-test/collections/system-management.postman_collection.json

```diff
@@ -251,7 +234,7 @@
           "raw": ""
         },
         "url": {
-          "raw": "{{systemManagementUrl}}/api/v1/metrics/device-virtual,edgex-core-command,edgex-core-data,edgex-core-metadata,edgex-export-client,edgex-export-distro,edgex-support-logging,edgex-support-notifications,edgex-support-scheduler",
+          "raw": "{{systemManagementUrl}}/api/v1/metrics/edgex-core-command",
```

Comment:
This converts the test from gathering metrics from multiple services to a single service. Is this intentional?

Is there a separate test that gathers metrics from more than one service?

Reply:
The intent was to split the `metrics`-related testing into two components: (1) `metrics` for a single service, and (2) `metrics` for all services. This has been addressed 👍